### PR TITLE
Upgrading the service framework version to the latest.

### PR DESCRIPTION
**Pull request by buchi-busireddy**

### CODE REVIEW

**Comment by buchi-busireddy** [comment]
Please hold off on the review of this. Raised https://github.com/hypertrace/service-framework/pull/12 to fix possible NPE when `app:<value>` tag is missing for a service. IntegrationTests are failing due to that same reason.
After merging that PR, i'll update this PR with new build.

**Review by kotharironak — APPROVED** [review]

**Comment by buchi-busireddy** [comment]
> Please hold off on the review of this. Raised [hypertrace/service-framework#12](https://github.com/hypertrace/service-framework/pull/12) to fix possible NPE when `app:<value>` tag is missing for a service. IntegrationTests are failing due to that same reason.
> After merging that PR, i'll update this PR with new build.

This is ready for review now.

**Pull request merged**
